orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/utils/HealthHelper.groovy

```diff
@@ -96,22 +98,19 @@ class HealthHelper {
     }
 
     // no health indicators is indicative of being down
-    boolean someAreDown = !healths || healths.any { it.state == 'Down' || it.state == 'OutOfService' || it.state == 'Starting' }
-    boolean noneAreUp = !healths.any { it.state == 'Up' }
+    boolean someAreDown = !healths || healths.any { HealthState.fromString(it.state) in [Down, OutOfService, Starting] }
```

Comment:
nit: these `.any` operations are now much more expensive and create more GC gen 0 objects.
Not sure how often this gets called. But might be worth making a new `healths` list that is just 
```java
healths.map { HealthState.fromString(it.state) }
```

and then check against that

Reply:
I considered it, keep in mind that most instances will only have 2 or 3 healths so I don't think it would change much

Reply:
ideally we would be working with a model from the get go as opposed to parsing strings from maps in here 😢 

I'll make a pass at a nicer cleanup later